Firebase/Auth/Source/FIRAuth.m

```diff
@@ -23,18 +23,14 @@
 #import "AuthProviders/EmailPassword/FIREmailPasswordAuthCredential.h"
 #import "AuthProviders/Phone/FIRPhoneAuthCredential_Internal.h"
 #import "Private/FIRAdditionalUserInfo_Internal.h"
-#import "Private/FIRAuthAPNSToken.h"
-#import "Private/FIRAuthAPNSTokenManager.h"
 #import "Private/FIRAuthAppCredentialManager.h"
```

Comment:
FIRAuthAppCredentialManager is only used for Phone number auth, so please move it to the inside of `#if TARGET_OS_IOS` as well.